modules/capacity_factors_wind_pv/workflow/Snakefile

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+from snakemake.utils import min_version, validate
+
+# Limit the snakemake version to a modern one.
+min_version("8.10")
+
+
+# Load the default configuration. This will be overridden by users.
+configfile: "config/default.yaml"
+
+
+# Validate the configuration using the schema file.
+# validate(config, workflow.source_path("schemas/config.schema.yaml"))
```

Comment:
Please add a fully restrictive schema for user config!
Otherwise, users have no way of knowing if they are configuring this module correctly.